tests/unit/core/testErrorHandler.js

```diff
@@ -263,5 +263,15 @@ describe('errorHandler', () => {
       const handler = new ErrorHandler();
       expect(handler.hasError()).toBe(false);
     });
+
+    it('returns no component if it does not have an error', () => {
+      const handler = new ErrorHandler();
+      expect(handler.renderErrorIfPresent()).toBe(null);
+    });
+
+    it('returns a component if it has an error', () => {
+      const handler = new ErrorHandler({ capturedError: new Error() });
+      expect(handler.renderErrorIfPresent()).not.toBe(null);
```

Comment:
Mmmh, but the `find()` method call must be done on a Enzyme wrapped component if I understand correctly, and `renderErrorIfPresent()` returns a plain old React component.